src/Confluent.Kafka/Impl/NativeMethods/NativeMethods.cs

```diff
@@ -742,7 +748,7 @@ long least_significant_bits
                 /* rd_kafka_ConfigEntry_t * */ IntPtr entry);
 
         [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
-        internal static extern IntPtr rd_kafka_ConfigEntry_is_synonym (
+        internal static extern IntPtr rd_kafka_ConfigEntry_is_synonym(
```

Comment:
Can we remove the changes done by style fix at places which we don't intend to touch? It's unnecessarily increasing the diff size